schema.sql

```diff
@@ -32,3 +32,10 @@ CREATE TABLE vets(id SERIAL PRIMARY KEY,name VARCHAR(255),age INTEGER,date_of_gr
 CREATE TABLE specializations( vets_id INT REFERENCES vets(id),species_id INT REFERENCES species(id));
 
 CREATE TABLE visits(vet_id INT REFERENCES vets(id),animal_id INT REFERENCES animals(id),date_of_visit date);
+
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX visits_animal_id ON visits(animal_id);
+CREATE INDEX visits_vet_id ON visits(vet_id);
+CREATE INDEX owner_email_index ON owners(email);
```

Comment:
- [ ] Nice work finding a solution to improve the preformation of the database :clap: However it is requested in the project requirement to add screenshots for each query **AFTER** and **BEFORE** the `explain analyze` to the PR.

Reply:
@LionRouge1  Done making the changes.